vcstool/clients/vcs_base.py

```diff
@@ -71,6 +74,64 @@ def _create_path(self):
                 }
         return None
 
+    def _create_or_truncate_path(self):
+        def remove_file(f):
+            try:
+                os.remove(f)
+            except OSError as e:
+                if getattr(e, 'winerror', None) == 5:
+                    # on Windows you need to clear the readonly bit first
+                    os.chmod(f, stat.S_IWRITE)
```

Comment:
I am hesitant to unconditionally change file attributes. The scope of this should be narrowed down to the specific case where it is desired (the `.git` directory).

Reply:
If other files have the `readonly` attribute, why should a `import --force` fail instead of deleting the file? I think it's more expected that, like with Linux, if the file can be deleted, it will be.